### PR TITLE
SSCS-9334- Notification – Stop the responseReceived going out for digital cases

### PR DESCRIPTION
**Pull request by ryandac**
SSCS-9334- Notification – Stop the responseReceived going out for digital cases

At the moment if DWP reply to a digital case and within the journey they select “Yes” for Upload contains further information to assist the tribunal (with the case being UC and requires an urgent hearing), then two sets of notifications are sent one for dwpUploadResponse and the other for responseReceived – this is due to both events having notifications linked to them. We want to keep the events, as responseReceived is used to commit case data changes, however we should probably add a block for the responseReceived going out.

### CODE REVIEW

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java [hunk 1]
```diff
@@ -177,12 +113,6 @@
     @Value("${notification.welsh.appealCreated.appointee.emailId}")
     private String appealCreatedAppointeeEmailIdWelsh;
 
-    @Value("${notification.welsh.hearingAdjourned.appointee.emailId}")
```
Comment (srochani) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java:180]: Why we hae to remove the hearingAdjouned one here?

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java [hunk 2]
```diff
@@ -219,9 +149,6 @@
     @Value("${notification.welsh.hearingBooked.joint_party.smsId}")
     private String jointPartyHearingBookedSmsIdWelsh;
 
-    @Value("${notification.welsh.hearingPostponed.appointee.emailId}")
```
Comment (srochani) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java:222]: same here

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java (hunk 1 above)
Comment (ryandac) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java:180]: It was an un-used variable.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java (hunk 2 above)
Comment (ryandac) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java:222]: It was an un-used variable.

**Review by srochani — APPROVED** [review]

**Pull request merged**